python/vision_explanation_methods/explanations/common.py

```diff
@@ -247,11 +247,11 @@ def expand_class_scores(
     """
     number_of_detections = scores.shape[0]
 
-    expanded_scores = torch.ones(number_of_detections, number_of_classes)
+    expanded_scores = torch.ones(number_of_detections, number_of_classes + 1)
```

Comment:
why did this file need to be changed?  it looks like expanded_scores and residual were changed.